mindjunkies/courses/tests/test_views.py

```
@@ -0,0 +1,118 @@
+import pytest
+from model_bakery import baker
+from django.urls import reverse
+from categories.models import Category
```

Ruff (0.8.2)

`categories.models.Category` imported but unused

Remove unused import: `categories.models.Category`

(F401)